frontend/packages/console-dynamic-plugin-sdk/src/api/core-api.ts

```diff
@@ -1,6 +1,26 @@
 /* eslint-disable */
 import { UseActivePerspective } from '../extensions/console-types';
 
+/**
+ * Hook that provides the currently active perspective and a callback for setting the active perspective
+ * @returns Tuple `[activePerspective, setActivePerspective]` where:
+ *  `activePerspective` is the currently active perspective as defined in redux state
+ *  `setActivePerspective` is a callback that accepts a string argument and sets the appropriate active perspective redux state
```

Comment:
Note that newlines aren't preserved, so I'd expect these to run together on one line.